segmentation/run_model.py

```diff
@@ -87,11 +89,14 @@ def get_segmentation_data(self, sample):
         volumes = sample[self.image_key_name]
         volumes = to_var(volumes[torchio.DATA].float(), self.device)
 
-        if self.label_key_name in sample:
-            targets = sample[self.label_key_name]
-            targets = to_var(targets[torchio.DATA].float(), self.device)
-        else:
-            targets = None
+        targets = []
+        for key in self.label_key_name:
+            if key in sample:
+                target = sample[key]
+                targets.append(to_var(target[torchio.DATA].float(), self.device))
+            else:
+                return volumes, None
```

Comment:
there should be at least a warning in the log,
en fait pas ici, car sinon on va avoir un message a chaque iteration, mais ce serait bien d'ajouter ce check dans le config, une fois qu'on a construit le dataset (donc on a la list des modalite,) vérifie que la liste des label_key_name correspond, si non, ajout un self.log('WARNING misssing ....')

Reply:
En fait j'ai mis ça car j'utilise le même getter pour l'inférence mais les labels ne sont pas présents à ce moment-là. Je devrais peut-être avoir quelque chose de plus explicite effectivement

Reply:
ca me va comme ca, 
mais je voulais juste qu'on prevoit le cas ou l'utilisateur fait une faute de frappe dans le nom label_key (dans le fichier json), mais c'est annecdotique